server/server/views/location_event.py

```diff
@@ -1,18 +1,33 @@
 from cornice import Service
-
 from ..models.event import Event
 
+from datetime import datetime, timedelta
 
 events_short_info = Service(name='events_short_info',
                             path='/events-short-info',
                             cors_origins=('http://localhost:3000',))
 
 
-@events_short_info.get()
+@events_short_info.post()
 def get_events_short_info(request):
     """ """
-    responce = {
+    datetime_now = datetime.now()
+    categories = {'holiday': 1, 'concert': 2, 'movie': 3, 'Voluntary': 4}
```

Comment:
Hardcoding is bad, we have get_all() method in the category model, you can use it.